code/modules/paperwork/photography.dm

```diff
@@ -343,10 +330,35 @@
 	name = "Old Camera"
 	desc = "An old, slightly beat-up digital camera, with a cheap photo printer taped on. It's a nice shade of blue."
 	icon_state = "oldcamera"
-	icon_on = "oldcamera"
-	icon_off = "oldcamera_off"
 	pictures_left = 30
 
+/obj/item/device/camera/broadcasting
+	name = "Broadcasting Camera"
+	desc = "Actively document everything you see, from the mundanity of shipside to the brutal battlefields below. Has a built in printer for action shots."
+	icon_state = "broadcastingcamera"
+	pictures_left = 20
+	pictures_max = 20
+	w_class = SIZE_HUGE
+	flags_equip_slot = NO_FLAGS //cannot be equiped
+	flags_item = TWOHANDED
+	item_state = "broadcastingcamera"
+	item_icons = list(
+		WEAR_L_HAND = 'icons/mob/humans/onmob/items_lefthand_0.dmi',
+		WEAR_R_HAND = 'icons/mob/humans/onmob/items_righthand_0.dmi'
+		)
+
+/obj/item/device/camera/broadcasting/attack_self(mob/user) //wielding capabilities
+	. = ..()
+	if(!(flags_item & TWOHANDED))
+		return
+	if(flags_item & WIELDED)
```

Comment:
Shouldn't need this `TWOHANDED` check here, since it won't change on the broadcasting camera and there aren't any subtypes.
```suggestion
	. = ..()
	if(flags_item & WIELDED)
```